apps/bfd-pipeline/bfd-pipeline-app/pom.xml

```diff
@@ -83,8 +83,15 @@
 
 		<dependency>
 			<!-- Used to run our unit and integration tests. -->
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<!-- Allows junit compatibility with old junit4 tests -->
+			<groupId>org.junit.vintage</groupId>
+			<artifactId>junit-vintage-engine</artifactId>
+			<version>${junit.vintage.version}</version>
```

Comment:
if we do end up merging this with this dependency, please put a comment explaining that we have it specifically (and exclusively) for the `@Rule` functionality

Reply:
Removed it, since I converted the remaining tests that needed it!